dev/release/04-binary-download.sh

```diff
@@ -36,4 +36,4 @@ crossbow_job_prefix="release-${version_with_rc}"
 : ${CROSSBOW_JOB_NUMBER:="0"}
 : ${CROSSBOW_JOB_ID:="${crossbow_job_prefix}-${CROSSBOW_JOB_NUMBER}"}
 
-archery crossbow download-artifacts ${CROSSBOW_JOB_ID}
+archery crossbow download-artifacts ${CROSSBOW_JOB_ID} --no-fetch
```

Comment:
It only works if the job from submitted from the same machine - which should be true in most of the cases.

Reply:
We can revisit this when this causes a problem.

For example, we can use `CROSSBOW_ARGS` environment variable to customize this behavior:

```bash
: ${CROSSBOW_ARGS:=--no-fetch}
```